docs/general/run-a-node/upgrade-log.md

```diff
@@ -239,21 +239,37 @@ spin up an Oasis Node in archive mode and query the pre-Damask state.
 :::
 
 7. Perform any needed [configuration changes][damask-conf-changes] described
-  below.
+   below.
 
-8. Start your node and re-enable auto-starting via your process manager.
+8. (only for ParaTime operators) Replace old versions of your ParaTime binaries
+   with the new ones.
```

Comment:
Maybe this sentence could mention the Oasis Runtime Containers (to avoid possible confusion, as there are no direct binaries in the new releases).

Something like: "Replace old versions of your ParaTime binaries
   with the updated Oasis Runtime Containers (`.orc`)."

Probably can come up with something better.

Reply:
Thanks for the suggestion. I've updated the text.